solr/solrj/src/java/org/apache/solr/client/solrj/impl/BaseCloudSolrClient.java

```diff
@@ -994,12 +991,15 @@ public RouteException(ErrorCode errorCode, NamedList<Throwable> throwables, Map<
           // and we could not get any information from the server
           //it is probably not worth trying again and again because
           // the state would not have been updated
-          log.info("trying request again");
+          log.info("Request to collection {} failed due to ({}) {}, retry={} maxRetries={} commError={} errorCode={} - retrying",
+              inputCollections, errorCode, rootCause, retryCount, MAX_STALE_RETRIES, wasCommError, errorCode);
           return requestWithRetryOnStaleState(request, retryCount + 1, inputCollections);
         }
       } else {
         log.info("request was not communication error it seems");
       }
+      log.error("Request to collection {} failed due to ({}) {}, retry={} maxRetries={} commError={} errorCode={} ",
```

Comment:
```suggestion
      log.info("Request to collection {} failed due to ({}) {}, retry={} maxRetries={} commError={} errorCode={} ",
```

@noblepaul - is this what you had in mind when you wrote "switch this to INFO" on the https://issues.apache.org/jira/browse/SOLR-14828 ticket? Or would you prefer keeping the logging at line 973 only but as `INFO` instead of `ERROR`?